### PR TITLE
Updates needed for launch of sponsor outreach

### PR DESCRIPTION
**Pull request by lesleychard**
- **Top Nav:** Remove 2021 Contest link, will add 2023 program link when page is ready, 2021 page still accessible via footer
- **Home Page:** Updated 2023 copy, improved desktop styles, removed old "Our Team" profiles
- **Sponsor Us Page:** Updated 2023 copy, more economic tiers, removed 2021 sponsors, updated FAQ
- **Volunteer Page:** Changed date from 2021 to 2023

TO DO: Add link to Sponsor Information Package when finalized this week.

### CODE REVIEW

**Review by Freddie-Pike — CHANGES_REQUESTED** [review]
Code reviewed, manually tested. This is looking awesome!!! 😄 Left a few minor comments. One thought I had when manually testing the PR is in the description you mention that "TODO: Add link to Sponsor Information Package when finalized this week" and when you click on the "Sponsor Us" button in the "Calling all sponsers" `RetroUI` component you get navigated to the new page. Do you want to have that page navigate to the old component for now, or is it okay for that button to exist for now?

**Comment by Freddie-Pike** [comment]
There's merge conflicts with my changes I merged into main, but they should be easy to fix as it's in `next.config.js` and you can fix it by accepting your changes and then adding back these environment variables:

```
MAILCHIMP_API_KEY: process.env.MAILCHIMP_API_KEY,
MAILCHIMP_LIST_ID: process.env.MAILCHIMP_LIST_ID,
```

File: layout/TopNav.tsx [hunk 1]
```diff
@@ -33,13 +37,19 @@ export const NAV_ITEMS: NavItem[][] = [
   ],
 ];
 
+console.log(getSiteVersion());
```
[the remark this answers is not in the document]
Comment (lesleychard) [reply layout/TopNav.tsx:40]: LMAO woops

File: modules/sponsor/SponsorFAQ.tsx [hunk 2]
```diff
@@ -41,15 +41,15 @@ const FAQ_ITEMS = [
...
     answer: [
-      'In the event we receive an overwhelming amount of community support, all extra donations will be carried over to the STEMforGirls programme to help fund other STEM events and learning opportunities for girls and gender diverse youth.',
+      'In the event we receive an overwhelming amount of community support, all extra donations will be carried over to further Digital Waves programs.',
     ],
   },
   {
     question: 'How can I sponsor this initiative all year round?',
     answer: [
-      'If you are interested in supporting girls & gender-diverse youth after the Digital Waves experience, please consider donating to one of our official partners, WRDC STEMforGirls or WISE Atlantic.',
+      'If you are interested in supporting girls & gender-diverse youth after the Digital Waves experience, please consider donating to one of our official partner, STEMforGIRLS',
```
[the remark this answers is not in the document]
Comment (lesleychard) [reply modules/sponsor/SponsorFAQ.tsx:52]: Editing text while eyes are burning 😵‍💫

File: pages/sponsor.tsx [hunk 3]
```diff
@@ -10,7 +10,6 @@ import FooterCTA from '../modules/footer/FooterCTA';
 import Footer from '../modules/footer/Footer';
 import FooterSubscribe from '../modules/footer/FooterSubscribe';
 import SocialMeta from '../components/SocialMeta';
-import SponsorPartners from '../modules/sponsor/SponsorPartners';
```
[the remark this answers is not in the document]
Comment (lesleychard) [reply pages/sponsor.tsx:13]: I don't think so - we will need to resurrect it again w/ new sponsors as soon as we start outreach :)

File: next.config.js [hunk 4]
```diff
@@ -6,6 +6,6 @@ module.exports = {
   env: {
     MAILCHIMP_SUBSCRIBE_URL: process.env.MAILCHIMP_SUBSCRIBE_URL,
     EMAILJS_USER_ID: process.env.EMAILJS_USER_ID,
-    NEXT_PUBLIC_SITE_VERSION: undefined,
+    NEXT_PUBLIC_SITE_VERSION: process.env.NEXT_PUBLIC_SITE_VERSION,
```
[the remark this answers is not in the document]
Comment (lesleychard) [reply next.config.js:9]: Up to you! I figured it wouldn't work unless it was set like this - but I can revert it back to `undefined`

**Comment by lesleychard** [comment]
> Code reviewed, manually tested. This is looking awesome!!! 😄 Left a few minor comments. One thought I had when manually testing the PR is in the description you mention that "TODO: Add link to Sponsor Information Package when finalized this week" and when you click on the "Sponsor Us" button in the "Calling all sponsers" RetroUI component you get navigated to the new page. Do you want to have that page navigate to the old component for now, or is it okay for that button to exist for now?

Yes! This button should go to the sponsor page, the Sponsor Information Package download will just be added to the Sponsor Us page.

File: pages/sponsor.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Freddie-Pike) [reply pages/sponsor.tsx:13]: Ahh, good point!

File: next.config.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (Freddie-Pike) [reply next.config.js:11]: Ahhh, you are def right, whoops! I'm sometimes a bit too safe when it comes to these things 😛

**Review by Freddie-Pike — APPROVED** [review]
Code reviewed, manually tested. This is awesome! Stoked to get the sponsor page up and running again 😄

**Pull request merged**